src/BatteryStateOfChargeService.h

```diff
@@ -36,14 +36,20 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     virtual ~BatteryStateOfChargeService();
 
     double totalAmpHoursUsed() const;
+    /*Promises: Returns the total amp hours used*/
+
     bool isCharging() const;
+    /*Promises: Returns through if the battery is being charged, false if not.*/
+
     QTime timeWhenChargedOrDepleted() const;
+    /*Promises: Return the time remaining to when the battery is charged or depleted*/
```

Comment:
Let's not use the "promises", just an explanation. Also generally comments go above a function.